jupyterlab/run.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env sh
 
+if [ -n "$BASE_URL" ]
```

Comment:
```suggestion
if [ -n "${BASE_URL}" ]
```
I think, by default, the `$BASE_URL` will be empty for both cases as `terraform` only templates when it is `${BASE_URL}`.

Reply:
Yes you're right